README.md

```diff
@@ -30,6 +37,12 @@ My Robot Test Case
     test code goes here
 ```
 
+## Parallel execution
+For parallel execution with e.g. `pabot`, you need to set parallel execution flag to `true`.<br>
+Listener uses synchronization through shared file for test cycle key and lock file - so only one process \ thread will create TM4J test cycle for execution, write its key into a shared file, and all other processes \ threads read test cycle key from it.<br>
+Unfortunately, Robot API doesn't have any methods that called after ALL executions are done, so you need to delete shared test cycle key file and lock file manually after run. The best way is to create some post-execution script that will do this cleanup for you.<br>
```

Comment:
Let's advise to delete shared flaf **before** parallel run, for example, in separate pre-execution script.